src/screens/home/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { Grid, Link } from '@material-ui/core'
+import { useLocation } from 'react-router-dom'
+import { PaperCard, PaperCardType } from './components/PaperCard'
+
+export default () => {
```

Comment:
We use to add a component Name (Home in this case) and after export it.

Reply:
Done